index.js

```diff
@@ -109,6 +113,24 @@ module.exports = (input, opts) => {
 		]);
 	}
 
+	if (runChecks) {
```

Comment:
`runGithubChecks`

Reply:
It’s `runStatusChecks` now (due to our discussion in https://github.com/sindresorhus/np/pull/289#discussion_r241930214). Should I still change it?

Reply:
Nah, it's good.